### PR TITLE
Add integration testing

### PR DESCRIPTION
**Pull request by simonrankine**
This will fail CI, and should not be merged until:

https://github.com/bbc/rd-apmm-groovy-ci-library/pull/38
https://github.com/bbc/nmos-joint-ri/pull/6

have been merged.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,7 +7,7 @@
  - Run Python 3 unit tests in tox
  - Build Debian packages for supported Ubuntu versions
 
- If these steps succeed and the master branch is being built, wheels and debs are uploaded to PyPI and the
+ If these steps succeed and the master branch is being built, wheels and debs are uploaded to Artifactory and the
```
Comment (andrewbonney) [inline Jenkinsfile]: Is there a merge issue here? We certainly were uploading this one to PyPI... Similar issues elsewhere in the file

File: Jenkinsfile [hunk 2]
```diff
@@ -78,6 +81,48 @@ pipeline {
...
+                            }
+                            stages{
+                                stage ("Start Test Environment") {
+                                    steps{
+                                        sh 'rm -r nmos-joint-ri || :'
+                                        withBBCGithubSSHAgent{
+                                            sh 'git clone -b simonra-integration-testing git@github.com:bbc/nmos-joint-ri.git'
+                                        }
+                                        dir ('nmos-joint-ri/vagrant') {
+					                        sh 'vagrant up --provision'
```
Comment (andrewbonney) [inline Jenkinsfile]: Large whitespace?

**Comment by simonrankine** [comment]
Should be in a position where this can be merged once the changes to the Groovy library are merged in.

**Comment by simonrankine** [comment]
Jenkins run CI

**Pull request merged**